docs/api/toolbar.md

```diff
@@ -24,941 +17,76 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
+Toolbars can contain several different components including titles, buttons, icons, back buttons, menu buttons, searchbars, segments, progress bars, and more.
 
 
-Toolbars are positioned above or below content. When a toolbar is placed in an `<ion-header>` it will appear fixed at the top of the content, and when it is in an `<ion-footer>` it will appear fixed at the bottom. Fullscreen content will scroll behind a toolbar in a header or footer. When placed within an `<ion-content>`, toolbars will scroll with the content.
+## Basic Usage
 
+It is recommended to put a toolbar inside of a [header](./header) or [footer](./footer) for proper positioning. When a toolbar is placed in a header it will appear fixed at the top of the content. When it is placed in a footer it will appear fixed at the bottom. Fullscreen content will scroll behind a toolbar in a header or footer. A [title](./title) component can be used to display text inside of the toolbar.
 
-## Buttons
+import Basic from '@site/static/usage/toolbar/basic/index.md';
 
-Buttons placed in a toolbar should be placed inside of the `<ion-buttons>` element. The `<ion-buttons>` element can be positioned inside of the toolbar using a named slot. The below chart has a description of each slot.
+<Basic />
 
-| Slot         | Description                                                                                              |
-|--------------|----------------------------------------------------------------------------------------------------------|
-| `secondary`  | Positions element to the `left` of the content in `ios` mode, and directly to the `right` in `md` mode.  |
-| `primary`    | Positions element to the `right` of the content in `ios` mode, and to the far `right` in `md` mode.      |
-| `start`      | Positions to the `left` of the content in LTR, and to the `right` in RTL.                                |
-| `end`        | Positions to the `right` of the content in LTR, and to the `left` in RTL.                                |
 
+## Buttons in Toolbars
 
-## Borders
+Buttons placed in a toolbar should be placed inside of the [buttons](./buttons) component. The buttons component can be positioned inside of the toolbar using a named [slot](#slots). The `"primary"` and `"secondary"` slots behave differently in `ios` and `md` mode. The buttons component can wrap a standard [button](./button), [back button](./back-button), a [menu button](./menu-button), or several of any of them. A button in a toolbar is styled to be clear by default, but this can be changed using the [`fill`](./button#fill) property on the button. Back buttons are linked to the app's navigation and will only show up when there is a previous page in the navigation stack, but we can bypass this by setting the `defaultHref` property. The menu button is set to only show when a linked menu is active, but this can be overridden by setting the `autoHide` property to `false`.
```

Comment:
```suggestion
Buttons placed in a toolbar should be placed inside of the [buttons](./buttons) component. The buttons component can be positioned inside of the toolbar using a named [slot](#slots). The `"primary"` and `"secondary"` slots behave differently in `ios` and `md` mode.

The buttons component can wrap a standard [button](./button), [back button](./back-button), a [menu button](./menu-button), or several of any of them. A button in a toolbar is styled to be clear by default, but this can be changed using the [`fill`](./button#fill) property on the button.
```
The descriptions of how `back-button` and `menu-button` work aren't needed since those should be covered by the respective component docs. (Also, the line break is optional but I think it makes this part easier to read 😄)

Reply:
Thanks! So I added that explanation in for the properties in menu button & back button because I had to use that in the code usage to avoid adding navigation or a menu to the examples. I thought adding all of that would be too much for the toolbar usage but wanted the back button & menu button to still show:

```
<ion-toolbar>
  <ion-buttons slot="start">
    <ion-back-button default-href="#"></ion-back-button>
  </ion-buttons>
  <ion-title>Back Button</ion-title>
</ion-toolbar>

<ion-toolbar>
  <ion-buttons slot="start">
    <ion-menu-button auto-hide="false"></ion-menu-button>
  </ion-buttons>
  <ion-title>Menu Button</ion-title>
</ion-toolbar>
```

Basically I was trying to let users know that they don't need to add this to their apps. Maybe there is a better way to get this point across? It might be bad to have because of people copy/pasting it without knowing why it's there.

Reply:
Maybe something like "for details on the properties used on `back-button` and `menu-button`, see their respective docs"? Reworded a little to be able to include inline links without it feeling awkward 😆 I just want to avoid duplicate info in case we change the props later.